packages/react-app-revamp/hooks/useContest/index.ts

```diff
@@ -180,9 +180,9 @@ export function useContest() {
       setIsListProposalsLoading(false);
 
       await Promise.all([
```

Comment:
Should probably have these components in a loading state until this is done - it looks like fetchTotalVotesCast at least shows as 0 for a second even in prod before it loads in.

Reply:
yeah that one should be independent as well, it is right now, but loading state is missing.

i'm up to create a separate PR for that change

Reply:
Dope, thank you!